lib/hotwire/spark/action_cable/server.rb

```diff
@@ -0,0 +1,7 @@
+class Hotwire::Spark::ActionCable::Server < ActionCable::Server::Base
+  def initialize(config: nil)
```

Comment:
The param is unnecessary. Removed in https://github.com/basecamp/hotwire-spark/commit/c10cbbeeca33a1a0fb06dfc5b3b146cb422e9ed4